### DISCUSSION TITLE
Ctrl-v cannot directly open a new split in the filepicker.

### DISCUSSION THREAD

**Discussion by suncunqian** [helix-editor/helix#11796] [Troubleshooting]
### Summary

In the filepicker interface, "Ctrl-s" can be triggered, but when I press "Ctrl-v", it only pastes some text into the input box.

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

WSL-debian

### Terminal Emulator

PowerShell

### Installation Method

release page

### Helix Version

helix 24.7 (079f5442)

**Comment by the-mikedavis** [comment]
Ctrl-v is handled either by your operating system or terminal emulator to paste. If you can't rebind Ctrl-v so it doesn't paste then you won't be able to use that keybinding for the picker. (That keybinding is not changeable in Helix currently, see #5505)
